apps/dog-walking-app/app/globals.css

```
@@ -0,0 +1,16 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
```

Stylelint (17.13.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)